Bottom: 57d7a3eca05bba75098d9b3c615218c1facfe4a7
Top:    3c0c3a13b66f4820c764a8b303a9e702a790de24
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-23 23:53:30 +0000

Refresh of remove-remaining-rcs-id-identi

---

diff --git a/src/parse_l.l b/src/parse_l.l
index 33a115d..c0ec6a7 100644
--- a/src/parse_l.l
+++ b/src/parse_l.l
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 %{
 /*
  *                            COPYRIGHT
@@ -48,9 +46,6 @@
 # include <dmalloc.h> /* see http://dmalloc.com */
 #endif
 
-RCSID("$Id$");
-
-
 #include "global.h"
 #include "crosshair.h"
 #include "data.h"
diff --git a/src/parse_y.y b/src/parse_y.y
index 8cf1888..adc2387 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -1,4 +1,3 @@
-/* $Id$ */
 /*
  * ************************** README *******************
  *
@@ -60,8 +59,6 @@
 # include <dmalloc.h> /* see http://dmalloc.com */
 #endif
 
-RCSID("$Id$");
-
 static	LayerTypePtr	Layer;
 static	PolygonTypePtr	Polygon;
 static	SymbolTypePtr	Symbol;
diff --git a/src/res_lex.l b/src/res_lex.l
index 5e08666..dd4dadd 100644
--- a/src/res_lex.l
+++ b/src/res_lex.l
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 %{
 
 #ifdef HAVE_CONFIG_H
@@ -18,8 +16,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID("$Id$");
-
 #define YY_NO_INPUT
 #define YY_INPUT(buf,result,max_size) { result = res_parse_getchars(buf, max_size); }
  
diff --git a/src/res_parse.y b/src/res_parse.y
index efc44a0..97c1aae 100644
--- a/src/res_parse.y
+++ b/src/res_parse.y
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 %{
 
 #ifdef HAVE_CONFIG_H
@@ -26,9 +24,6 @@
 #include <dmalloc.h>
 #endif
 
-
-RCSID("$Id$");
-
 static Resource *parsed_res;
 static Resource *current_res;
